performanceplatform/collector/piwik/query.py

```diff
@@ -0,0 +1,95 @@
+import re
+from datetime import datetime
+import pytz
+
+from performanceplatform.collector.piwik.base import BaseFetcher
+from performanceplatform.utils.data_pusher import Pusher
+from performanceplatform.utils.data_parser import DataParser
+
+FREQUENCY_TO_PERIOD_MAPPING = {
+    'daily': 'day',
+    'weekly': 'week',
+    'monthly': 'month',
+}
+
+
+class Fetcher(BaseFetcher):
+    def __init__(self, credentials, query, start_at, end_at):
+        super(Fetcher, self).__init__(credentials, query)
+        self.period = FREQUENCY_TO_PERIOD_MAPPING[
+            query.get('frequency', 'weekly')]
+        self.date = Fetcher.set_piwik_date(start_at, end_at)
+        self.api_method_arguments = query.get('api_method_arguments', None)
+
+    @classmethod
```

Comment:
I think these should be `@staticmethod` as they dont use the cls variable. Or, in python its fine to have methods that are not part of classes